mantidimaging/gui/windows/recon/view.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, main_window: MainWindowView):
 
         self.algorithmName.insertItem(1, "FBP_CUDA")
         self.algorithmName.insertItem(2, "SIRT_CUDA")
-        self.algorithmName.insertItem(3, "CIL: PDHG-TV")
+        self.algorithmName.insertItem(3, "CIL_PDHG-TV")
```

Comment:
'CIL_PDHG_TV' to be consistent

Reply:
Same as above

Reply:
Same as above, happy to modify based on you're opinion to my comment above